tests/System.Time.Tests/DateTimeOffsetTests.cs

```diff
@@ -7,10 +7,17 @@ namespace System.Time.Tests
 {
     public class DateTimeOffsetTests
     {
+
+#if Windows
+    string TimeZoneId = "Pacific Standard Time";
+#else
+    string TimeZoneId = "America/Los_Angeles";
```

Comment:
You should try running these tests on Mac. Are these strings Linux specific, or UNIX (and therefore Mac). Otherwise, you might need more change here to cover Mac.